build_and_test.sh

```diff
@@ -21,89 +24,52 @@ fi
 echo "==========================================================="
 echo "Test OpenFAM with cis-rpc-meta-direct-mem-rpc configuration"
 echo "==========================================================="
-export OPENFAM_ROOT="$MEMSRV_BUILD_DIR/test/config_files/config-cis-rpc-meta-direct-mem-rpc/"
-source setup.sh --cisserver=127.0.0.1  --memserver=127.0.0.1
-echo $OPENFAM_ROOT
-make unit-test
-if [[ $? > 0 ]]
-then
-        echo "One or more unit-test failed while running with cis-rpc-meta-direct-mem-rpc configuration. exit..."
-        exit 1
-fi
-make reg-test
```

Comment:
you can still keep one example of how we are starting the services manually and running each test, for developers it is important to have this setup for running and developing unit tests.